addons/website/static/src/scss/website.scss

```diff
@@ -1400,3 +1400,11 @@ $ribbon-padding: 100px;
         bottom: 0;
     }
 }
+
+.o_website_btn_loading {
+    cursor: wait;
+    opacity: 0.5;
```

Comment:
To not break bootstrap/theme style I would use the disabled class on top of this one.

Reply:
I am not too sure to understand, do you mean the JS code should add `disabled` class? That's already the case, this is just a 'security' that is barely noticeable on top of the disabled class but is needed for 'Edit' (top right) to mimick the btn disable style as the 'Edit' is not a btn.
Just a nice to have

Reply:
> that is barely noticeable

Yeah... you know that would not be a good argument against me ^^ Maybe with standard bootstrap style but if someones / some theme configures the disabled button colors (this is a BS variable), you don't expect it to be altered by a opacity: 0.5 that you will definitely see.

Since it is to support a special case, I would not fix that with a generic rule: add a rule on that fake button style to tell it what to do when being disabled. `.my_edit_button.disabled { ... } `